brownie/network/transaction.py

```diff
@@ -615,6 +615,49 @@ def info(self) -> None:
                     result += f"\n      {key}: {color('bright blue')}{value}{color}"
         print(result)
 
+    def _get_trace_gas(self, start: int, stop: int) -> Tuple[int, int]:
+        total_gas = 0
+        internal_gas = 0
+        is_internal = True
+        trace = self.trace
+
+        is_gas_forwarded = trace[start]["depth"] > trace[start - 1]["depth"]
```

Comment:
What if `start == 0` ?

Reply:
You are right, `start = 0` will compare the first to the last element. Since the last and first step always have the same depth and "from 0" is never an external call, this would work, but I added a check for `start > 0` to be more precise and a also inlined this variable, since it ended up being used only once.